README.md

```diff
@@ -1 +1,161 @@
-# botwo
\ No newline at end of file
+# Boto S3 Router
+
+Interact with S3-compatible services side-by-side with S3, without making any changes to your code. This package provides a Boto3-like client that routes requests between S3 clients according to the bucket and the key in the request.
+
+## Why use Boto S3 Router
+
+Consider the following code, downloading an object from one bucket and uploading it to another after some manipulation:
+
+```python
+def calculate(s3_client):
+    obj = s3_client.get_object(Bucket="bucket1", Key="a/b/c/obj")
+    obj2 = do_something_to_obj(obj)
+    s3_client.put_object(Body=obj2, Bucket="bucket2", Key="a/b/c/obj")
+```
+
+Suppose you want to migrate only `bucket2` to an S3-compatible storage like lakeFS or MinIO. 
```

Comment:
I think that it is confusing to say that lakeFS is an S3-compatible **storage**

Reply:
Changed to "service"?